CMSIS/Core/Include/r-profile/cmsis_iccarm_r.h

```diff
@@ -0,0 +1,171 @@
+/**************************************************************************//**
+ * @file     cmsis_armclang_r.h
+ * @brief    CMSIS compiler armclang (Arm Compiler 6) header file
+ * @version  V6.0.0
+ * @date     04. December 2024
+ ******************************************************************************/
+/*
+ * Copyright (c) 2009-2023 Arm Limited. All rights reserved.
```

Comment:
Would you want add IAR copyright here as well?

Reply:
Please change date. 04. December 2024 is not yet reached.
Please change Copyright year to 2009-2024

Reply:
Things went a little bit quick here, just wanted to get a draft out of the door.

> Would you want add IAR copyright here as well?

Sure 

> Please change date. 04. December 2024 is not yet reached.

Hm. This is actually based on https://github.com/ARM-software/CMSIS_6/blob/main/CMSIS/Core/Include/r-profile/cmsis_armclang_r.h and should be fixed there as well.

Reply:
Probably this is a typo, and the year actually was 2023 not 2024.

Reply:
@GuentherMartin, didn't we remove these (dublicated info) from the file headers, anyway? I.e., only the copyright shall be kept, but no file versions and dates?

Reply:
I would suggest to remove date and version items from the header in upcoming changes.

Reply:
The file header has been removed from all include file except:
  cmsis_armclamg.h, cmsis_gcc.h, cmsis_clang.h
  cmsis_armclamg_r.h, cmsis_gcc_r.h, cmsis_clang_r.h

I do not know why we kept the file headers in these files. We should remove them.

Reply:
I think that happened during the refactoring/reordering of the files. So, let's remove the headers in upcoming changes, as you suggested.

Reply:
Fixed!